centreon/cypress/fixtures/resources/newDetailsAnomalyDetection.json

```diff
@@ -0,0 +1,80 @@
+{
```

Comment:
What's the different with the previous one?
Besides that, the correct naming should be: anomalyDetectionDetails and not the other way around

Reply:
there is a difference in sensitivity object , the newDetailsAnomalyDetection , this is the result recieved after the user changes the size of envelop  (so different values of sensitivity object) , i check if the value is well updated while the user changed the envelop size 🤔